sympy/printing/mathml.py

```diff
@@ -861,6 +865,28 @@ def _print_binomial(self, expr, exp=None):
 
     def _print_Pow(self, e):
         # Here we use root instead of power if the exponent is the reciprocal of an integer
+        if self._settings['fold_short_frac'] and len(str(e.base)) < 7:
+            if e.exp.is_negative:
+                frac = self.dom.createElement('mfrac')
+                frac.setAttribute('bevelled', 'true')
+                frac.appendChild(self._print(1))
+                mrow = self.dom.createElement('mrow')
+                if e.exp.p == -1 and e.exp.q == 1:
+                    mrow.appendChild(self._print(e.base))
+                    frac.appendChild(mrow)
+                    return frac
+                if type(e.base) == symbol.Symbol:
```

Comment:
I realized that you can do `e.base.is_symbol` (correct syntax would otherwise by `e.base is Symbol`).

Reply:
Ohh Yeah. It's working.